docs/swagger.yaml

```diff
@@ -378,6 +376,16 @@ components:
           - image/svg+xml
           - text/targetUrl
       description: 'Requested image format. Supported values: {image/*,image/png, image/svg+xml, text/targetUrl}'
+    Size:
+      name: Size
+      in: header
+      required: false
+      schema:
+        type: integer
+        pattern: '^([1-9][0-9]{2}|[1-9][0-9]{3}|2000)$'
```

Comment:
You can remove this if you want, the autogenerated code will add filter for range(100, 2000) as long as a minimum and maximum value is set :) 
But, doesnt really matter 
cc @Sigvedash